Microsoft.Xrm.Data.PowerShell/Microsoft.Xrm.Data.PowerShell.psm1

```diff
@@ -3943,6 +3943,74 @@ function Invoke-CrmWhoAmI{
     return $result
 }
 
+function Invoke-CrmAction {
+# .ExternalHelp Microsoft.Xrm.Data.PowerShell.Help.xml
+    [OutputType([hashtable])]
+    [OutputType([Microsoft.Xrm.Sdk.OrganizationResponse], ParameterSetName="Raw")]
+    param (
+        [Microsoft.Xrm.Tooling.Connector.CrmServiceClient]
+        $conn,
+
+        [Parameter(
+            Position=1,
+            Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Name,
+
+        [Parameter(ValueFromPipeline)]
+        [ValidateNotNullOrEmpty()]
+        [Microsoft.Xrm.Sdk.EntityReference]
+        $Target,
+
+        [Parameter(Position=2)]
```

Comment:
The $Target parameter is optional, and to call the function without it but still have it be positional didn't seem to be possible. I think I tried to make a ParameterSetName to try and assign it a position, but it was ambiguous because all the other parameters were still being used in both parameter sets.